parachain/pallets/outbound-queue/src/mock.rs

```diff
@@ -0,0 +1,177 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+use super::*;
+
+use frame_support::{
+	parameter_types,
+	traits::{Everything, Hooks},
+	weights::IdentityFee,
+};
+
+use snowbridge_core::outbound::*;
+use sp_core::{ConstU32, H160, H256};
+use sp_runtime::{
+	traits::{BlakeTwo256, IdentityLookup, Keccak256},
+	AccountId32, BuildStorage,
+};
+
+type Block = frame_system::mocking::MockBlock<Test>;
+type AccountId = AccountId32;
+
+frame_support::construct_runtime!(
+	pub enum Test
+	{
+		System: frame_system::{Pallet, Call, Storage, Event<T>},
+		MessageQueue: pallet_message_queue::{Pallet, Call, Storage, Event<T>},
+		OutboundQueue: crate::{Pallet, Storage, Config<T>, Event<T>},
+	}
+);
+
+parameter_types! {
+	pub const BlockHashCount: u64 = 250;
+}
+
+impl frame_system::Config for Test {
+	type BaseCallFilter = Everything;
+	type BlockWeights = ();
+	type BlockLength = ();
+	type RuntimeOrigin = RuntimeOrigin;
+	type RuntimeCall = RuntimeCall;
+	type Hash = H256;
+	type Hashing = BlakeTwo256;
+	type AccountId = AccountId;
+	type Lookup = IdentityLookup<Self::AccountId>;
+	type RuntimeEvent = RuntimeEvent;
+	type BlockHashCount = BlockHashCount;
+	type DbWeight = ();
+	type Version = ();
+	type PalletInfo = PalletInfo;
+	type AccountData = ();
+	type OnNewAccount = ();
+	type OnKilledAccount = ();
+	type SystemWeightInfo = ();
+	type SS58Prefix = ();
+	type OnSetCode = ();
+	type MaxConsumers = frame_support::traits::ConstU32<16>;
+	type Nonce = u64;
+	type Block = Block;
+}
+
+parameter_types! {
+	pub const HeapSize: u32 = 32 * 1024;
+	pub const MaxStale: u32 = 32;
+	pub static ServiceWeight: Option<Weight> = Some(Weight::from_parts(100, 100));
+}
+
+impl pallet_message_queue::Config for Test {
+	type RuntimeEvent = RuntimeEvent;
+	type WeightInfo = ();
+	type MessageProcessor = OutboundQueue;
+	type Size = u32;
+	type QueueChangeHandler = ();
+	type HeapSize = HeapSize;
+	type MaxStale = MaxStale;
+	type ServiceWeight = ServiceWeight;
+	type QueuePausedQuery = OutboundQueue;
+}
+
+parameter_types! {
+	pub const OwnParaId: ParaId = ParaId::new(1013);
+}
+
+impl crate::Config for Test {
+	type RuntimeEvent = RuntimeEvent;
+	type Hashing = Keccak256;
+	type MessageQueue = MessageQueue;
+	type MaxMessagePayloadSize = ConstU32<1024>;
+	type MaxMessagesPerBlock = ConstU32<20>;
+	type OwnParaId = OwnParaId;
+	type GasMeter = ();
```

Comment:
Prefer to use `ConstantGasMeter` rather than the unit struct as close to the production config as possible.

Reply:
Fixed in https://github.com/Snowfork/snowbridge/pull/982/commits/a0d7916624012f2b32c4c6ce846e8ba092c6d5bb